src/ipfs.io/docs/getting-started/index.html

```diff
@@ -122,7 +122,7 @@
 Now, you should be able to get objects from the network. Try:
 
 ```
-ipfs cat /ipfs/QmW2WQi7j6c7UgJTarActp7tDNikE4B2qXtFCfLPdsgaTQ/cat.jpg >cat.jpg
```

Comment:
no, this is supposed to be `ipfs cat`. notice the stdout redirection with `>cat.jpg`